### DISCUSSION TITLE
request to improve user instructions in readme.md

### DISCUSSION THREAD

**Discussion by jacquesbg** [burghardt/clamfs#95] [General]
I have difficulties to get clamfs running properly, because I do not understand the flow of the program. 
I think the problem is that I do not know what 
`<filesystem root= ... />` and `<filesystem mountpoint= ...  />`
are needed for.
Maybe an example use case will help. In my case I want that all files that are written to (or read from  (?)) the ~/Downloads folder are scanned. What would for this case be the proper values for `<filesystem root= ... />` and `<filesystem mountpoint= ...  />`?

Thanks.

**Comment by burghardt** [comment]
Simpliest way to do it on Debian:
```
sudo apt install fuse3 clamfs
zcat /usr/share/doc/clamfs/clamfs-sample.xml.gz > ~/.clamfs-downloads.xml
mv ~/Downloads ~/.Downloads-raw/
mkdir ~/Downloads
```

Now edit configuration file `~/.clamfs-downloads.xml` and set the following:
```
<filesystem root="/home/user/.Downloads-raw" mountpoint="/home/user/Downloads" public="no" />
```

Finally start ClamFS and verify that EICAR causes FS denying read:
```
clamfs ~/.clamfs-downloads.xml
# download something like EICAR
file *
eicar.com:            writable, regular file, no read permission
````

And the log should be similar to:
```
Aug 18 18:46:55 disp8737 clamfs[10854]: (file:11013) (user:1000) '/home/user/.Downloads-raw/eicar.com': fd[10]: Eicar-Signature FOUND
```

HTH

**Comment by jacquesbg** [comment]
Thanks, that works.
